src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -74,16 +75,26 @@ function ButtonWithDropdownMenu<IValueType>({
         }
         if ('measureInWindow' in dropdownAnchor.current) {
             dropdownAnchor.current.measureInWindow((x, y, w, h) => {
+                let horizontalPosition;
```

Comment:
NAB Suggestion: We can define `let horizontalPosition = x + w;` here and remove the else